packages/frontend/src/lib/StatusIcon.svelte

```diff
@@ -1,9 +1,10 @@
 <script lang="ts">
+import type { knownStatus } from './StatusIcon';
 // status: one of RUNNING, STARTING, USED, CREATED, DELETING, or DEGRADED
 // any other status will result in a standard outlined box
 import Spinner from '/@/lib/button/Spinner.svelte';
 
-export let status = '';
+export let status: knownStatus | string = '';
```

Comment:
not sure why we need the "else string".

Reply:
the doc indicates that the component can accept any other value:
```
// any other status will result in a standard outlined box
```